Snippets/Rabbit/Rabbit_All/Rabbit_All.csproj

```diff
@@ -1,15 +1,22 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <TargetFrameworks>net462</TargetFrameworks>
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>
+
   <ItemGroup>
+    <ProjectReference Include="..\..\Common\Common.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.*" />
```

Comment:
FMI, how was this working before without this?

Reply:
Without that, the tests weren't showing up in VS at at all.